Completion/Unix/Command/_email-notmuch

```diff
@@ -0,0 +1,9 @@
+#autoload
+
+local expl
+local -a notmuch_addr
+
+notmuch_addr=( ${(f)"$(notmuch address --deduplicate=address --output=address -- from:/$PREFIX/)"} )
```

Comment:
Does $PREFIX need to be shell- or regexp-escaped?

Please look into `_call_program` and use it where needed, to allow overriding the commands using zstyle's.